rmw/include/rmw/types.h

```diff
@@ -239,6 +289,154 @@ typedef enum RMW_PUBLIC_TYPE
   RMW_LOG_SEVERITY_FATAL = RCUTILS_LOG_SEVERITY_FATAL
 } rmw_log_severity_t;
 
+/// Defined in the DDS SSpec 15-04-10 section 2.2.4.1 Communication Status
+typedef struct RMW_SAMPLE_LOST_STATUS
```

Comment:
typedef struct <CAPS> does not follow conventions
https://google.github.io/styleguide/cppguide.html#Type_Names